test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/SetInteriorVehicleData/018_Change_audio_source_in_FULL_rc_non_media_app.lua

```diff
@@ -0,0 +1,67 @@
+---------------------------------------------------------------------------------------------------
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case:
+-- 1) App is RC and Non-Media
+-- 2) App in any HMI level
+-- 3) App tries to change audio source
+-- SDL must:
+-- 1) Not change audio source
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.isMediaApplication = false
+
+--[[ Local Variables ]]
+local audioData = common.getModuleControlData("AUDIO")
+local audioSources = {
+  "NO_SOURCE_SELECTED",
+  "CD",
+  "BLUETOOTH_STEREO_BTST",
+  "USB",
+  "USB2",
+  "LINE_IN",
+  "IPOD",
+  "MOBILE_APP",
+  "RADIO_TUNER"
+}
+
+--[[ Local Functions ]]
+local function setVehicleData(pSource)
+  audioData.audioControlData.source = pSource
+  local cid = common.getMobileSession():SendRPC("SetInteriorVehicleData", {
+      moduleData = audioData
+    })
+
+  EXPECT_HMICALL("RC.SetInteriorVehicleData")
+  :Times(0)
+
+  -- Confirmation nedded, not clear what resultCode must be
+  common.getMobileSession():ExpectResponse(cid, { success = false, resultCode = "REJECTED" })
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("RAI, PTU", common.rai_ptu_n)
```

Comment:
@GetmanetsIrina Please note in ```common.rai_ptu_n()``` it is used default ```PTUfunc()``` func. This will lead to change app's HMI type to RC.

Reply:
@dboltovskyi , application has RC HMI type, app is RC application and non-media.

Reply:
@GetmanetsIrina Ok - thank you for explanation